src/app.module.ts

```diff
@@ -3,11 +3,12 @@ import { AppController } from './app.controller';
 import { AppService } from './app.service';
 import { AuthModule } from './modules/auth/auth.module';
 import { UserModule } from './modules/users/user/user.module';
+import { ConfigModule } from './modules/config/config.module';
 
 
 
```

Comment:
remove empty lines.
Should only have 1 line break between `import` section and module section